ui/package.json

```diff
@@ -151,38 +167,27 @@
     "ember-cli-get-component-path-option": "^1.0.0",
     "ember-cli-string-utils": "^1.1.0"
   },
+  "packageManager": "pnpm@10.13.1+sha512.37ebf1a5c7a30d5fabe0c5df44ee8da4c965ca0c5af3dbab28c3a1681b70a256218d05c81c9c0dcf767ef6b8551eb5b960042b9ed4300c59242336377e01cfad",
   "engines": {
-    "node": "16.* || 18.* || 20.*"
+    "node": "16.* || 18.* || 20.*",
+    "pnpm": ">= 10"
+  },
+  "pnpm": {
+    "overrides": {
+      "@babel/runtime@<7.26.10": ">=7.26.10",
```

Comment:
this is magic to me -- is this saying "if [something] tries to install this dependency below this version, then instead reverse that and make sure _above_ that version gets installed/used" ?

Reply:
this particular section is where we enforce and mitigate some of the security issues on translative dependancies, I will reflect that in the description as well